db/init/postgrest.sql

```diff
@@ -112,11 +112,12 @@ CREATE OR REPLACE FUNCTION enermaps_query_geojson(parameters text,
                     jsonb_object_agg(variable, value) as variables,
                     jsonb_object_agg(variable, unit) as units,
                     fields,
+                    layer,
```

Comment:
What is this new variable?

Reply:
This is a json field containing information on the type of the layer so as to build a color bar/legend for categorical datasets. By default `{"type": "numerical"}` otherwise with classes, e.g.   {`"type": "categorical", "classes": {0: "warmer climate", 1: "average climate", 2: "colder climate"}}`